synapse/storage/databases/main/relations.py

```diff
@@ -514,14 +504,15 @@ def _get_thread_summaries_txn(
                         AND parent.room_id = child.room_id
                     WHERE
                         %s
-                        AND relation_type = ?
+                        AND (relation_type = ? OR relation_type = ?)
```

Comment:
This (and the similar changes) mean that having both `m.thread` and `io.element.thread` in a thread would work...for now (until support for `io.element.thread` is dropped).

I prefer doing it this way though since it does not double the queries to get stable and unstable threads. Thoughts on whether this is acceptable?

Reply:
The above is still true, but only if you have the experimental config flag enabled.